Tools/BatchScripts/batch_summit.sh

```diff
@@ -25,5 +25,8 @@ umask 0027
 # disable all the IBM optimized barriers and drop back to HCOLL or OMPI's barrier implementations
 export OMPI_MCA_coll_ibm_skip_barrier=true
 
+# ROMIO has a hint for GPFS named IBM_largeblock_io which optimizes I/O with operations on large blocks
```

Comment:
Yes, let's do this then because users copy from here.

Can we add this for simplicity via environment variables, instead of adding an auxiliary file for `ROMIO_HINTS`? :) 
Otherwise, I would say we create the hints file here automatically with bash's `echo ... >`, so that users do not need to worry (and it's then just there for `jsrun`).

Reply:
Yep, done!